### PR TITLE
Replaced Google Analytics with Matomo

### PR DESCRIPTION
**Pull request by rehrar**
Replaced Google Analytics with an open-source, self-hosted alternative: Matomo.

Some cleanup in the footer include also.

### CODE REVIEW

**Comment by philiparthurmoore** [comment]
@rehrar Just curious, where are the `piwik` files located? Is that just an unversioned server installation that's staying out of the repo? I see them referenced in the commit (`js` and `php`). Thanks a ton.

**Comment by rehrar** [comment]
It's on a server managed and maintained by the Core Team, similar to getmonero.org. What would be your recommendation for making this more transparent?

**Comment by philiparthurmoore** [comment]
@rehrar This is perfect. I've made mistakes in the past implementing self-hosted analytics and wanted to understand the code. :-)

**Comment by MaxXor** [comment]
Well done! :+1:

File: _includes/footer.html [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
-                           {% for item in site.data.lang[site.lang].footer-2 %}
-                            {% if item.url contains 'https://' %}
-                             <li><a href="{{item.url}}" class="white footer-link">{{item.name}}</a></li>
-                            {%else%}
-                             <li><a href="{{site.baseurl}}/{{item.url}}" class="white footer-link">{{item.name}}</a></li>
-                           {%endif%}
-                           {% endfor %}
-                        </ul>
-                    </div>
+    <div class="container">
```
Comment (leonklingele) [inline _includes/footer.html]: Why was this change necessary? Please revert those lines.

**Comment by rehrar** [comment]
Blerg, I screwed something up and it did something else also. :/ Sorry, let me revert these.

**Comment by rehrar** [comment]
Alright, @leonklingele take a look now and tell me how it goes (hint: I probably screwed it up :P). Thanks for your help.
